src/test/shared/schema/assertions.ts

```diff
@@ -0,0 +1,50 @@
+/*!
```

Comment:
This file can go in `schema/testUtils`.

Hyper-granularity harms discoverability by adding noise (which makes the signal harder to find). It's very unlikely that we will need these things in other test files--and if we ever do, it's easy to split things out (analysis is easier than synthesis).